frontend/server/src/Controllers/Identity.php

```diff
@@ -558,6 +558,31 @@ public static function apiChangePassword(\OmegaUp\Request $r): array {
         ];
     }
 
+    /**
+     * Entry point for change account of a session
+     *
+     * @throws \OmegaUp\Exceptions\InvalidParameterException
+     *
+     * @return array{auth_token: string}
+     *
+     * @omegaup-request-param string $usernameOrEmail
+     */
+    public static function apiChangeAccount(\OmegaUp\Request $r) {
```

Comment:
```suggestion
    public static function apiSelectIdentity(\OmegaUp\Request $r) {
```

usamos el verbo select en más lugares. y la inclusión de "identity" en el nombre deja más claro a qué se refiere.

Reply:
Listo